dotnet/samples/KernelSyntaxExamples/Example60_FlowOrchestrator.cs

```diff
@@ -0,0 +1,336 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.AI;
+using Microsoft.SemanticKernel.AI.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.AI.OpenAI;
+using Microsoft.SemanticKernel.Experimental.Orchestration;
+using Microsoft.SemanticKernel.Orchestration;
+using Microsoft.SemanticKernel.Plugins.Core;
+using Microsoft.SemanticKernel.Plugins.Memory;
+using Microsoft.SemanticKernel.Plugins.Web;
+using Microsoft.SemanticKernel.Plugins.Web.Bing;
+using NCalcPlugins;
+
+/**
+ * This example shows how to use FlowOrchestrator to execute a given flow with interaction with client.
+ */
+
+// ReSharper disable once InconsistentNaming
+public static class Example60_FlowOrchestrator
+{
+    private static readonly Flow s_flow = FlowSerializer.DeserializeFromYaml(@"
+name: FlowOrchestrator_Example_Flow
+goal: answer question and send email
+steps:
+  - goal: What is the tallest mountain on Earth? How tall is it divided by 2?
+    plugins:
+      - WebSearchEnginePlugin
+      - LanguageCalculatorPlugin
+    provides:
+      - answer
+  - goal: Collect email address
+    plugins:
+      - ChatPlugin
+    completionType: AtLeastOnce
+    transitionMessage: do you want to send it to another email address?
+    provides:
+      - email_addresses
+
+  - goal: Send email
+    plugins:
+      - EmailPluginV2
+    requires:
+      - email_addresses
+      - answer
+    provides:
+      - email
+
+provides:
+    - email
+");
+
+    public static Task RunAsync()
+    {
+        // Load assemblies for external plugins
+        Console.WriteLine("Loading {0}", typeof(SimpleCalculatorPlugin).AssemblyQualifiedName);
+
+        return RunExampleAsync();
+        //return RunInteractiveAsync();
```

Comment:
I enabled this line of code (and commented out line 67) and ran the sample.

My ask was "What is the length of the longest river in ireland?"

The sample seemed to go into a loop and eventually failed with this error:
{"Failed to complete step 0_What is the length of the longest river in ireland?_0 for session 3ad41b1e-76bb-4eab-bd94-73a1ece37123."}

Reply:
It is because of missing the `PromptTemplateEngine`, I think it just became mandatory for our extension. I will fix the kernel bootstrap in tests in next commit